smack-extensions/src/main/java/org/jivesoftware/smackx/muc/UserStatusListener.java

```diff
@@ -123,4 +123,12 @@
      */
     public abstract void adminRevoked();
 
+    /**
+     * Called when the room is destroyed.
+     * 
+     * @param alternateMUC an alternate MultiUserChat.
```

Comment:
IIRC alternateMUC and reason are both optional. If so, it should be mentioned in the javadoc that they may be null.